bin/plugin-server

```diff
@@ -22,10 +22,9 @@ while test $# -gt 0; do
 done
 
 export BASE_DIR=$(dirname $(dirname "$PWD/${0#./}"))
-# If KAFKA_URL is set, extract netlocs into KAFKA_HOSTS by removing "kafka://" in a compatibility approach
-if [ -n "$KAFKA_URL" ]; then
-  export KAFKA_HOSTS=$(echo $KAFKA_URL | sed -e "s/kafka\(\+ssl\)\{0,1\}:\/\///g")
```

Comment:
If we do want to fix this regex it should be:
`sed -E "s/kafka((\+ssl)?):\/\///g"`

Reply:
This wasn't working before, so merging this in, will unblock release and if this is needed somewhere we can follow up